src/pages/HelpSupport/components/ContactPage/ContactPage.tsx

```diff
@@ -1,79 +1,90 @@
-import { FC, CSSProperties } from 'react'
-import Button from '@/components/Button'
+import { useState, FC } from 'react'
+import { BiEnvelope, BiPhone, BiChat } from 'react-icons/bi';
+import Button from '@/components/Button';
+import ChatboxPage from '../ChatBox/ChatboxPage';
```

Comment:
remove never used code

Reply:
THX, Already removed line 3，but remian line 4， because line 76 of this file have a code (<ChatboxPage isOpen={isOpen} handleClose={handleClose} />) still need it

Reply:
And already fix the package.jason, I delete the code:
engine:{"node"}: 18.0.0, yarn:3.6.60}
and the code:
packagemanager: yarn 3.6.0

Reply:
I prefer the former one.

Reply:
"I prefer the former one." What does this mean？

Reply:
Please keep `engine:{"node"}: 18.0.0, yarn:3.6.60}` and remove `packagemanager: yarn 3.6.0`

Reply:
Sorry, I just tested this, if you run `yarn set version 3.6.0`, `packagemanager: yarn 3.6.0` will be added to `package.json`. Also add `yarnPath: .yarn/releases/yarn-3.6.0.cjs` to `.yarnrc.yml`

We can gitignore `.yarnrc.yml` file, and keep `packagemanager: yarn 3.6.0`

Reply:
老师，不好意思这几天生病了，才开始弄，我想问下这个是需要我做什么的意思吗